### PR TITLE
Add IRC guidelines

### PR DESCRIPTION
**Pull request by GregSutcliffe**
To be merged next week after the community has had a chance to review. Mail going to -users now.

### CODE REVIEW

**Comment by domcleal** [comment]
:+1: generally.  I find the text a bit long for the page, since it pushes mailing lists and the bug tracker down, but my only suggestions are:

a) a separate page
b) a hidden/expanding section

File: support.md [hunk 1]
```diff
@@ -24,7 +24,41 @@ passwords or tokens and create a tarball which can be safely handed over to
...
+Be aware that the IRC channel is not an official support channel, it's an ad
+hoc group of people (some of whom work on Foreman for a living) self-organizing
+to help each other out. If you do not receive an answer to your question,
+(especially if you have not followed the getting help on irc best practices!),
+that doesn’t mean you are out of options, the software is hopelessly broken, or
+your problem is insoluble. It just means you need to keep troubleshooting.
+
+We also have a few IRC-specific guidelines:
+
+* Don't be a jerk: Treat people with respect and consideration.
```
Comment (lzap) [inline support.md:50]: I'd maybe pull this a little bit down the list :-D

File: support.md [hunk 2]
```diff
@@ -24,7 +24,41 @@ passwords or tokens and create a tarball which can be safely handed over to
...
+* Be helpful: Be patient with new people and be willing to jump in to answer questions.
+* Stay calm: The written word is always subject to interpretation, so give people the benefit of the doubt and try not to let emotions get out of control.
+* Don't post chunks: Avoid posting big chunks of text - use a [pastebin](http://pastie.org/) or similar service to shorten it to a link. Likewise, try not to spam the channel with many messages, it breaks the flow of other conversations.
+* Be patient: Folks might not be around when you ask a question, so wait a while for someone to speak before leaving.
+* Search first: Believe it or not, your question might not be new or you might be able to find someone who has already asked or answered your question. Do a thorough search of the [Manual](http://theforeman.org/), the [Redmine Issues](http://projects.theforeman.org/projects/foreman/issues), and the [Mailing List](https://groups.google.com/forum/#!forum/foreman-users) to see if it has been answered before.
```
Comment (lzap) [inline support.md:55]: This should be the very first bullet! In Bold!

Comment (GregSutcliffe) [inline support.md:55]: I'd rather help someone who's being polite but perhaps hasn't searched very well, than someone who searched well but is being a jerk. IMO the order is fine :)

File: support.md [hunk 3]
```diff
@@ -24,7 +24,41 @@ passwords or tokens and create a tarball which can be safely handed over to
...
+ask to ask", and "Stick around".  #theforeman in particular has concentrations
+of people in Europe (CEST) and East-coast US (EST), so asking your question
+when those time zones are in business hours is more likely to get a good
+result.
+
+Be aware that the IRC channel is not an official support channel, it's an ad
+hoc group of people (some of whom work on Foreman for a living) self-organizing
+to help each other out. If you do not receive an answer to your question,
+(especially if you have not followed the getting help on irc best practices!),
+that doesn’t mean you are out of options, the software is hopelessly broken, or
```
Comment (Yamakasi) [inline support.md:45]: I would change: "the software is hopelessly broken" into "the software might have a bug (http://projects.theforeman.org/projects/foreman/issues), the docs are lacking and need a change or....

**Comment by GregSutcliffe** [comment]
@domcleal we could simply make the IRC the last section on the page so that all headings are initially visible?

**Comment by domcleal** [comment]
@GregSutcliffe good solution!

**Comment by domcleal** [comment]
Merge o'clock?  (once comments are integrated)

**Comment by GregSutcliffe** [comment]
:+1:

**Comment by GregSutcliffe** [comment]
Merged as d7e40e1, thanks everyone!

**Pull request closed without merging**